content/about.md

```diff
@@ -2,4 +2,195 @@
 title = "About Matrix"
 +++
 
-*To be filled with content*
+## Governance
+
+The evolution of Matrix is managed through an open governance process, looked
+after by The Matrix.org Foundation - ‡ non-profit UK Community Interest
+Company, incorporated to act as the neutral guardian of the standard on behalf
+of the whole Matrix community.
+
+The Foundation defines the manifesto, mission and values of the project, the
+open governance process that determines how the specification develops, and
+provides a safety-net to ensure the project stays independent and true to its
+goals. The constitution of the project is defined in the Foundation’s legal
+Articles of Association and Rules, and is enforced by the Guardians of Matrix:
+the directors of the Foundation appointed to provide a balance of independent
+experts and the founding Matrix team to ensure the project stays on track.
+
+### The Guardians
+
+The Guardians are the legal directors of the non-profit Foundation, and are
+responsible for ensuring that the Foundation (and by extension the Spec Core
+Team) keeps on mission and neutrally protects the development of Matrix.
+Guardians are typically independent of the commercial Matrix ecosystem and may
+even not be members of today’s Matrix community, but are deeply aligned with
+the mission of the project. Guardians are selected to be respected and trusted
+by the wider community to uphold the guiding principles of the Foundation and
+keep the other Guardians honest.
+
+In alphabetical order:
+
+TODO: do sort of cards, like for speakers at conferences.
```

Comment:
@jplatte what's the best way to do this? Should the content for each guardian be in their own markdown file, and the about page template should be updated to add it?

Is it possible to make the template iterate through all the files in a directory to add them?

Reply:
I don't think we can iterate through files in a directory, how about a toml file like we have for the frontpage (the sponsor listing)? https://github.com/matrix-org/matrix.org/blob/zola/frontpage.toml

Reply:
We can if the main page is a section but then each card has a subpage 🤔 don't think this is what we want.

Reply:
A toml file would probably be great! Can it be `content/about.toml`, or would it clash with content/about.md`?

Reply:
Should be no problem to have them in content. I'm happy moving `frontpage.md` to `content/index.md` as well if you prefer.